core/common/participant-context-config-core/src/main/java/org/eclipse/edc/participantcontext/config/ParticipantContextConfigServicesExtension.java

```diff
@@ -14,41 +14,53 @@
 
 package org.eclipse.edc.participantcontext.config;
 
-import org.eclipse.edc.encryption.EncryptionService;
+import org.eclipse.edc.encryption.EncryptionAlgorithmRegistry;
 import org.eclipse.edc.participantcontext.config.service.ParticipantContextConfigServiceImpl;
 import org.eclipse.edc.participantcontext.spi.config.ParticipantContextConfig;
 import org.eclipse.edc.participantcontext.spi.config.service.ParticipantContextConfigService;
 import org.eclipse.edc.participantcontext.spi.config.store.ParticipantContextConfigStore;
 import org.eclipse.edc.runtime.metamodel.annotation.Extension;
 import org.eclipse.edc.runtime.metamodel.annotation.Inject;
 import org.eclipse.edc.runtime.metamodel.annotation.Provider;
+import org.eclipse.edc.runtime.metamodel.annotation.Setting;
+import org.eclipse.edc.spi.monitor.Monitor;
 import org.eclipse.edc.spi.system.ServiceExtension;
 import org.eclipse.edc.transaction.spi.TransactionContext;
 
-import static org.eclipse.edc.participantcontext.config.defaults.ParticipantContextConfigDefaultServicesExtension.NAME;
+import static org.eclipse.edc.participantcontext.config.ParticipantContextConfigServicesExtension.NAME;
 
 @Extension(NAME)
 public class ParticipantContextConfigServicesExtension implements ServiceExtension {
 
     public static final String NAME = "Participant Context Config Services Extension";
 
+    @Setting(
+            description = "The encryption algorithm used for encrypting and decrypting sensitive config.",
+            key = "edc.participants.config.encryption",
```

Comment:
nit: maybe something like `edc.participants.config.encryption.algorithm` would be more intuitive?